project.clj

```diff
@@ -3,21 +3,20 @@
   :url "http://publishmydata.com"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
+  :plugins [[lein-tools-deps "0.4.1"]]
+  :middleware [lein-tools-deps.plugin/resolve-dependencies-with-deps-edn]
+  :lein-tools-deps/config {:config-files [:install :project]}
```

Comment:
Why remove `:user`?  It should normally be there so users can pick up their own aliases etc.